### DISCUSSION TITLE
关于FISM模型的损失函数和参数问题

### DISCUSSION THREAD

**Discussion by Liayja** [RUCAIBox/RecBole#1252] [Q&A] [answered]
在原论文中，作者提出了两种损失函数，分别是FISMrmse和FISMauc，因此，想问一下，框架中的FISM模型是怎么定义损失函数的呢？
在调整参数的时候，发现只有embedding_size、alpha和两个正则化权重（w1、w2）这些参数，是否是用到了FISMauc呢？

**Comment by Ethan-TZ** [comment, marked as answer]
@Liaoyejia 您好，感谢您的关注。我们使用的是BCE Loss或者说是Log Loss。我们的实现参考了NAIS paper作者给出的源码：[NAIS](https://github.com/AaronHeee/Neural-Attentive-Item-Similarity-Model)，同样使用的是Log Loss。

**Reply by Liayja** [reply]
好的，谢谢您的解答！
